src/ceph-disk/ceph_disk/main.py

```diff
@@ -3153,6 +3153,61 @@ def move_mount(
     )
 
 
+def systemd_enable(
+    path,
+    osd_id,
+):
+    # ensure there is no duplicate ceph-osd@.service
+    for style in ([], ['--runtime']):
+        command_check_call(
+            [
+                'systemctl',
+                'disable',
+                'ceph-osd@{osd_id}'.format(osd_id=osd_id),
+            ] + style,
```

Comment:
This is for upgrade purposes, to make sure there are no competing units. There can be two units at the same time: one with --runtime and another without it. If, for any reason (manual or ceph-disk) the two units co-exist they will compete with each other.